### DISCUSSION TITLE
Create react app clears the output of other projects in v2, used to work in v1

### DISCUSSION THREAD

**Discussion by clicktodev** [vercel/turborepo#9547] [Help] [answered]
### Summary

when i run npm run dev, multiple projects spin up but their output gets cleared with no way to scroll back up.
this wasn't the case in v1

you can see in the video how the output gets deleted quickly and only the last project output is visible whereas the other projects output was auto cleared

**Comment by anthonyshew** [comment, marked as answer]
This is the fault of `create-react-app` not handling terminal outputs correctly: https://github.com/facebook/create-react-app/issues/2495

This is one of the many reasons that we built [the new Terminal UI in 2.0](https://turbo.build/blog/turbo-2-0#new-terminal-ui). I'd highly encourage you to use it instead, since we can handle poorly behaved tools like these with this iteration of the UI.

**Reply by clicktodev** [reply]
Understandable but this wasn’t an issue in v1.
Did v1 implement some kind of workaround around the create react app behavior?

**Reply by anthonyshew** [reply]
If this ever worked with the streamed logs output, it was purely out of luck. We have no control over terminal outputs or what underlying tasks try to print in the streamed logs.

**Reply by clicktodev** [reply]
actually after some testing, i discovered this change was introduced in [1.12.0](https://github.com/vercel/turborepo/releases/tag/v1.12.0)
you can test it in this repo by setting the version to 1.12.0 https://github.com/vercel/react-tweet
